### DISCUSSION TITLE
Add support for netboot.xyz Enhancement Request

### DISCUSSION THREAD

**Discussion by jphein** [ltsp/ltsp#206] [Q&A]
Can we add netboot.xyz under ipxe more options? 
It works really well. Check here: https://github.com/ltsp/community/wiki/netboot.xyz-(ipxe-OS-installer)
Only problem is that my edits are reset everytime I run `ltsp ipxe`

**Comment by alkisg** [comment]
No, this is too specific; but, if in the future many people ask support for "custom ipxe entries", we might add these.

In the meantime, administrators that need a custom ltsp.ipxe can just run `ltsp ipxe` once and then edit it however they need.

**Comment by alkisg** [comment]
Hmm it's probably best to move WONTFIX issues to the community, and leave them open so that people can discuss about it, rather than just close them here. Moving...

**Comment by jphein** [comment]
Thanks @alkisg! I understand. 
Would the "custom ipxe entries" be added in ltsp.conf?
Like CUSTOM_IPXE_NETBOOTXYZ="chain --autofree https://boot.netboot.xyz" ?
I really like the idea, but don't NEED it at the moment.

**Comment by alkisg** [comment]
Right, something like that.

On the other hand, if ltsp.ipxe gets too many parameters and becomes too customizable, it won't be user-editable anymore, so there's a balance involved.
